phac-nml/irida-next
# Fix: Weird path generation behaviour

## What does this PR do and why?
_Describe in detail what your merge request does and why._

When a path is generated while creating a new Project/Group, dash characters were added in weird locations.

e.g. `BDIPtest` would slugify to `bdi-ptest`

This occurred due to a default option of `{decamelize: true}` in [`@sindresorhus/slugify`](https://www.npmjs.com/package/@sindresorhus/slugify#decamelize). Changing that option to `false` has resulted in more normal path generation, e.g. `BDIPtest` now slugifys to `bdiptest`.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Launch server
2. Attempt to create a new project with name `BDIPtest` and confirm that path is generated to `bdiptest`
3. Attempt to create a new group with name `TESTtestTESTtest` and confirm that path is generated to `testtesttesttest`

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
